tethys_gizmos/static/tethys_gizmos/js/tethys_map_view.js

```diff
@@ -1791,6 +1810,9 @@ var TETHYS_MAP_VIEW = (function() {
 
     var feature = map.forEachFeatureAtPixel(event.pixel,
         function(feature, layer) {
+          if (layer.editable === false){
+            return false;
```

Comment:
Should this return false or undefined? The other return statement returns an object, not a boolean.